changelog/2757.bugfix.rst

```diff
@@ -0,0 +1 @@
+Send http.client.connect audit events
```

Comment:
We use keepachangelog.com style changelogs, so let's start this newsfragment with "Fixed", ie:

> Fixed `urllib3.connection.HTTPConnection` to raise the `http.client.connect` audit event to have the same behavior as the standard library HTTP client".